.github/workflows/shared-increase-version-number.yaml

```diff
@@ -21,13 +21,17 @@ on:
       image_tag:
         description: The image tag used to extract metadata for Docker
         value: ${{ jobs.incrementVersionNumber.outputs.image_tag }}
+      git_tag_or_hash:
+        description: The git tag or hash (for snapshots) containing the updated version.
+        value: ${{ jobs.incrementVersionNumber.git_tag_or_hash }}
 
 jobs:
   incrementVersionNumber:
     runs-on: ubuntu-latest
     outputs:
       new_version: ${{ steps.version.outputs.new_version }}
       image_tag: ${{ steps.updatePackageJson.outputs.image_tag }}
+      git_tag_or_hash: ${{ steps.commit-and-tag.result == 'success' && steps.commit-and-tag.outputs.git_tag_or_hash || steps.commit-without-tag.outputs.git_tag_or_hash }}
```

Comment:
Using the success result so we don't need to keep the condition in sync with the condition that chooses which step to run.

Reply:
Where is `steps.commit-and-tag.result` defined? I checked the documentation and it seems it should be `steps.<step_id>.outcome`: https://docs.github.com/en/actions/learn-github-actions/contexts#steps-context